test/helpers/fixtures.js

```diff
@@ -1,22 +1,29 @@
 import db from '../../src/services/db';
-import * as baseSql from '../../src/base/sql';
+
+import pgp from '../../src/base/pgp';
 
 const data = {
   state: require('../fixtures/state')
 };
 
+function resetSequence () {
+  return Promise.all(Object.keys(data).map(table => {
+    return db.task(t => {
+      const tableName = new pgp.helpers.TableName(table);
+      return t.one('SELECT MAX(id) FROM $1', tableName).then(result => {
```

Comment:
This is totally a premature optimization, but the way you've written this requires a round trip with the database to get the max value when you could just use a subselect:
```sql
SELECT setval('${table}_id_seq', (SELECT MAX(id) FROM $1))
```